COSTouchVisualizer.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
```

Comment:
Did you verify this change should be made?

Reply:
Which change in particular?

Reply:
Setting the version an, homepage, and source?

Reply:
I think incrementing the version makes sense, as then we know that we're using our new version.

As for the other two, it feels like the right way to go. I don't think this file has a huge amount of bearing on anything at the moment – I think it's really for populating info on https://cocoapods.org/pods, but we're not submitting our repos there as far as I understand.

@multitracks Have you got a particular way you want us to go on this?

Reply:
This is great 👍